server/middlewares/validators/videos.ts

```diff
@@ -221,6 +223,47 @@ const videosShareValidator = [
   }
 ]
 
+const videosChangeOwnershipValidator = [
+  param('id').custom(isIdOrUUIDValid).not().isEmpty().withMessage('Should have a valid id'),
```

Comment:
I don't think this id could be an UUID?

Reply:
It is the video id. I used the same validations than other functions (like `videosUpdateValidator` in same file).

Reply:
Oh right, maybe `videoId` would be a better parameter name